apps/shared/avifutil.c

```diff
@@ -252,7 +252,7 @@ avifAppFileFormat avifReadImage(const char * filename,
             return AVIF_APP_FILE_FORMAT_UNKNOWN;
         }
     } else {
-        fprintf(stderr, "Unrecognized file format: %s\n", filename);
+        fprintf(stderr, "Unrecognized file format for input file: %s\n", filename);
```

Comment:
I believe `avifReadImage()` is only used with input files. If we are not sure, we can omit "input" in the error message.

Reply:
I think it's always an input file in the sense that it's a file we're reading. I thought the error message was clearer this way.